lib/miq_automation_engine/service_models/miq_ae_service_service.rb

```diff
@@ -9,7 +9,6 @@ class MiqAeServiceService < MiqAeServiceModelBase
     require_relative "mixins/miq_ae_external_url_mixin"
     include MiqAeExternalUrlMixin
 
-    expose :retire_service_resources
```

Comment:
We need to keep this `expose` as the backend method is called from content here: https://github.com/ManageIQ/manageiq-content/blob/master/content/automate/ManageIQ/Service/Retirement/StateMachines/Methods.class/__methods__/retire_service.rb#L5

This change will turn it into a no-op which is also incorrect.  The required logic was restored when the delete was reverted in https://github.com/ManageIQ/manageiq/pull/19417.

This PR should be closed.

Reply:
Wasn't it already a no-op?

Reply:
Yes, you are right.  Let me review all this with @tinaafitz today.